marc.go

```diff
@@ -244,8 +244,12 @@ func NewMarcIterator(r io.Reader) *MarcIterator {
 func makeDataField(tag string, data []byte) (DataField, error) {
 	d := DataField{}
 	d.Tag = tag
-	d.Indicator1 = string(data[0])
-	d.Indicator2 = string(data[1])
+	if len(data) > 0 {
```

Comment:
Should we actually check to make sure that it has at least 3 bytes of data? Just to avoid having to do this again when we come across a field with only one indicator.

Reply:
Probably. I wasn't really thinking about anything beyond "just @#!$ing work".